proxy/http2/HTTP2.cc

```diff
@@ -681,6 +686,7 @@ Http2::init()
   REC_EstablishStaticConfigInt32U(max_concurrent_streams_in, "proxy.config.http2.max_concurrent_streams_in");
   REC_EstablishStaticConfigInt32U(min_concurrent_streams_in, "proxy.config.http2.min_concurrent_streams_in");
   REC_EstablishStaticConfigInt32U(max_active_streams_in, "proxy.config.http2.max_active_streams_in");
+  REC_EstablishStaticConfigBool(stream_priority_enabled, "proxy.config.http2.stream_priority_enabled");
```

Comment:
You sure ConfigBool here is appropriate? I don't see us use it anywhere, and the configs in RecordsConfig.cc / records.config are always of integer type. Not opposed to using Bool's here, but seems a little bit inconsistent.